platform/android/dir_access_jandroid.cpp

```diff
@@ -135,17 +135,27 @@ String DirAccessJAndroid::get_drive(int p_drive) {
 	}
 }
 
+String DirAccessJAndroid::_get_root_string() const {
+	if (get_access_type() == ACCESS_FILESYSTEM) {
+		return "/";
+	}
+	return DirAccessUnix::_get_root_string();
+}
```

Comment:
I wonder if this shouldn't be handled in `DirAccessUnix`, currently it seems the only implementation is in `DirAccess` and it doesn't handle `ACCESS_FILESYSTEM` explicitly, it just returns `""`.

Reply:
I only saw a regression on the Android platform, so I limited the scope of the changes to that platform only to avoid introducing accidental regressions.

`DirAccessUnix` has a separate implementation of `get_current_dir()` which handles the empty root string.